Modules/Loadable/Markups/Widgets/qMRMLMarkupsAdditionalOptionsWidgetsFactory.h

```diff
@@ -23,65 +23,65 @@
 #define __qslicermarkupsfactory_h_
 
 // Markups widgets includes
-#include "qSlicerMarkupsAdditionalOptionsWidget.h"
+#include "qMRMLMarkupsAdditionalOptionsWidget.h"
 #include "qSlicerMarkupsModuleWidgetsExport.h"
 
 // Qt includes
 #include <QObject>
 #include <QList>
 #include <QPointer>
 
-class qSlicerMarkupsAdditionalOptionsWidget;
-class qSlicerMarkupsAdditionalOptionsWidgetsFactoryCleanup;
+class qMRMLMarkupsAdditionalOptionsWidget;
+class qMRMLMarkupsAdditionalOptionsWidgetsFactoryCleanup;
 
 /// \ingroup SlicerRt_QtModules_Segmentations
-/// \class qSlicerMarkupsAdditionalOptionsWidgetsFactory
+/// \class qMRMLMarkupsAdditionalOptionsWidgetsFactory
 /// \brief Singleton class managing segment editor effect plugins
-class Q_SLICER_MODULE_MARKUPS_WIDGETS_EXPORT qSlicerMarkupsAdditionalOptionsWidgetsFactory : public QObject
+class Q_SLICER_MODULE_MARKUPS_WIDGETS_EXPORT qMRMLMarkupsAdditionalOptionsWidgetsFactory : public QObject
 {
   Q_OBJECT
 
 public:
   /// Instance getter for the singleton class
   /// \return Instance object
-  Q_INVOKABLE static qSlicerMarkupsAdditionalOptionsWidgetsFactory* instance();
+  Q_INVOKABLE static qMRMLMarkupsAdditionalOptionsWidgetsFactory* instance();
 
 public:
   /// Registers an additional options widget.
   /// This factory object takes ownership of the widget (even if it fails to register it
   /// due to for example a widget already existing for that markup type), therefore the
   /// caller must not delete the widget.
-  Q_INVOKABLE bool registerAdditionalOptionsWidget(qSlicerMarkupsAdditionalOptionsWidget* widget);
+  Q_INVOKABLE bool registerAdditionalOptionsWidget(qMRMLMarkupsAdditionalOptionsWidget* widget);
 
   /// Unregisters an additional options widget.
-  Q_INVOKABLE bool unregisterAdditionalOptionsWidget(qSlicerMarkupsAdditionalOptionsWidget* widget);
+  Q_INVOKABLE bool unregisterAdditionalOptionsWidget(qMRMLMarkupsAdditionalOptionsWidget* widget);
```

Comment:
This appears to be not a "factory" just a storage of a few pre-created widgets. The factory should store function pointers or classes (preferable the latter) that can instantiate any number of the desired widgets.